src/app/dashboard/chat/ChatUI.tsx

```
@@ -1 +1,52 @@
-export { ChatUI } from "~/components/chat"; 
\ No newline at end of file
+import { ChatUI as InnerChatUI } from "~/components/chat";
+import 'highlight.js/styles/github.css';
+import { useChat } from 'ai/react';
+import { useCallback, useEffect } from 'react';
+import { api } from "~/trpc/react";
+import { toast } from "sonner";
+
+export function ChatUI(props: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)